sc-memory/sc-core/CMakeLists.txt

```diff
@@ -1,21 +1,15 @@
-set(SC_CORE_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
+set(SC_CORE_SRC ${CMAKE_CURRENT_SOURCE_DIR})
 
-file(GLOB_RECURSE SOURCES "*.c" "*.h" "*.cpp" "*.hpp")
+file(GLOB SOURCES CONFIGURE_DEPENDS
+    "*.c" "*.h"
+    "*/*.c" "*/*.h"
+    "*/*/*.c" "*/*/*.h"
+    "*/*/*/*.c" "*/*/*/*.h"
+)
```

Comment:
Would've been much nicer if we had CMake targets for sc-core submodules such as sc-store and sc-container, but that's just thinking out loud

Reply:
It had better to make it later. It's not important now.